architecture-decisions/0008-migration-from-dataspace.md

```diff
@@ -0,0 +1,99 @@
+
+#### DOIs
+
+If a work already has a DOI, it is important that we NOT register a new one. 
+
+<img src="images/enter_existing_doi.png"
+     alt="Screenshot of Create Dataset menu item"
+     style="margin-left: 10px; height: 200px" />
+
+1. If there is an existing DOI, enter it on the `Curator Controlled` tab. If this work is from PPPL, it likely has an existing DOI that is not in DataSpace. Go to the [osti website](https://www.osti.gov) to find it. Make sure to record the DOI for the Dataset, *not* the DOI for the publication.
+1.  You can also record the associated publication if you want to, as a "Related Object"
+2. Not every items in DataSpace will have a DOI. If you do not enter an existing DOI, the system will record a new one.  
+
+#### Creators
+1. Enter the creators of the work in publication order. Author order matters. 
+2. Enter an ORCID identifier for every creator you can find. There is [a spreadsheet of ORCIDs for PPPL](https://docs.google.com/spreadsheets/d/1U6AuWCLogVGBcNXmH4p6o8ZQc2nleyt0s0TedOpNkC0/edit#gid=0) researchers. If you cannot find an ORCID for a creator on a PPPL dataset, enter their name on [the second tab of the spreadsheet](https://docs.google.com/spreadsheets/d/1U6AuWCLogVGBcNXmH4p6o8ZQc2nleyt0s0TedOpNkC0/edit#gid=1142499854) and someome from PPPL will help you track it down.
+
+#### Description
+1. All variations on description are going into the same field. We are not going to distinguish between `abstract` and `description`, or other kinds of descriptive text.
+2. Remove all references to `Globus` from the description field. These will not be necessary in the new system, and they are not actually a description of this dataset. 
+
+#### Funders
+1. Where possible, identify the funders of the work that created this dataset, along with the Award Number and Award URI if known.
+2. In DataSpace, funders are often found listed as a `Contributor`. Sometimes they can also be found in the README file if that exists. 
+3. Where possible, record the Research Organization Registry (ROR) identifier for an organization. Consult [https://ror.org/](https://ror.org/).
+
+#### Other contributors
+1. Where possible, enter other contributors with their ORCIDs and RORs, if you can find them.
+
+#### Collection
+1. Works from PPPL should go into the `Princeton Plasma Physics` group. For PPPL items, also record the subcollection.
+2. Everything else goes into the PRDS group.
+
+#### ARK
+1. The ARK is the unique identifier from DataSpace that will allow us to migrate the data payload. Make sure to enter the ARK exactly how it appears in DataSpace. 
+
+#### Migrate the data
+1. Once your metadata is complete, you can click the "Migrate Data" button and the data from this work will be automatically moved from DataSpace to PDC Describe pre-curation. In some cases this might take a long time. You may need to put the work down for a day and come back to it after the migration process is complete. 
+
```

Comment:
```suggestion
#### Click Migrate not Save 
  To mark that this is a migrated work click the `Migrate` button on the Create Data form.  This marks the item as a migration item and enables the `Migrate Dataspace Files` button on the work show page.
  
#### Migrate the data
1. Once your metadata is complete, you can click the "Migrate Dataspace Files" button on the work show page and the data from this work will be automatically moved from DataSpace to PDC Describe pre-curation. In some cases this might take a long time. You may need to put the work down for a day and come back to it after the migration process is complete.  **Note:  The migration of data is queued in the background, so your browser will return after all files are queued, but before the file data has completed migration**

```